<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startwindow</class>
 <widget class="QWidget" name="Startwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Sign up
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please select your account type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OrganisationButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Organisation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PersonalButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnonymousButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anonymous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
